### PR TITLE
DFR-2278: FR CONSENT - caseworkers upload approved order

### PR DESCRIPTION
**Pull request by rishikrsharma**
### JIRA link (if applicable) ###
DFR-2278

### Change description ###
caseworkers upload approved order

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: definitions/consented/json/CaseEvent/CaseEvent.json [hunk 1]
```diff
@@ -1016,7 +1016,7 @@
     "Name": "Upload Approved Order",
     "Description": "Upload Approved Order",
     "DisplayOrder": 62,
-    "PreConditionState(s)": "referredToJudge",
+    "PreConditionState(s)": "referredToJudge;readyForHearing",
```
Comment (srochani) [inline definitions/consented/json/CaseEvent/CaseEvent.json:1019]: AC2 awaiting Judicial response state is not used here

Comment (srochani) [reply definitions/consented/json/CaseEvent/CaseEvent.json:1019]: Also AC3 is not covered here

Comment (rishikrsharma) [reply definitions/consented/json/CaseEvent/CaseEvent.json:1019]: In state Id do not match with name
  "ID": "referredToJudge",
    "Name": "Awaiting Judicial Response",

**Review by srochani — APPROVED** [review]

File: definitions/consented/json/CaseEvent/CaseEvent.json (hunk 1 above)
Comment (rishikrsharma) [reply definitions/consented/json/CaseEvent/CaseEvent.json:1019]: AC3 is already existing functionality

**Pull request merged**